src/ast/Node.hpp

```diff
@@ -124,6 +133,8 @@ Node Node::make(const Args&... args) {
 	static_assert(
 	    std::is_same<T, math::Real>::value ||
 	    std::is_same<T, math::Rational>::value ||
+		std::is_same<T, math::ComplexReal>::value ||
+		std::is_same<T, math::ComplexRational>::value ||
```

Comment:
omg, indentation is screwed up everywhere. Can you convert everything to spaces (in next PR or something)?